test/api/model/model_test.dart

```diff
@@ -581,6 +581,9 @@ void main() {
       );
       check(m2).flags.deepEquals([MessageFlag.read, MessageFlag.unknown]);
     });
+
+    // Code relevant to messageEditState is tested separately in the
+    // MessageEditState group.
```

Comment:
Alternatively, we could also nest those tests inside this group. That's basically what I did for the `UpdateMessageTest` groups in the next commit.

Reply:
I think having its own group at the end of the test file (like it is in this revision) is good. The code being tested is mainly in `MessageEditState`, which is at the end of its file, so having the tests at the end keeps the tests arranged in the same order as the code they're testing. That's helpful when navigating the tests.